src/main/lrsql/system/lrs.clj

```diff
@@ -328,4 +329,12 @@
       (jdbc/with-transaction [tx conn]
         (auth-cmd/delete-credential! backend tx cred-in)
         {:api-key    api-key
-         :secret-key secret-key}))))
+         :secret-key secret-key})))
+
+  adp/AdminStatusProvider
+  (-get-status
+    [this params]
+    (let [conn (lrs-conn this)]
+      (jdbc/with-transaction [tx conn]
+        (admin-q/query-status
+         backend tx (admin-stat-input/query-status-input params))))))
```

Comment:
Would probably be clearer if the input fn call was put in a let-binding.

Reply:
Yeah sure, reading the other methods that would be consistent. https://github.com/yetanalytics/lrsql/pull/282/commits/5a60033925e951a2e317a74f417b576be00ce905